tests/test_cliCmdDefineVisits.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for daf_butler CLI define-visits command.
+"""
+
+import click
+import click.testing
+import unittest
+
+from lsst.daf.butler.cli import butler
+from lsst.daf.butler.cli.utils import Mocker, mockEnvVar
+
+
+def makeExpectedKwargs(**kwargs):
+    expected = dict(config_file=None,
+                    collections=[])
+    expected.update(kwargs)
+    return expected
+
+
+class Case(unittest.TestCase):
```

Comment:
The convention is to name TestCases in a descriptive way so that test reports are more readable. This should be something like `CommandLineDefineVisitTestCase` or somesuch.